src/main/java/de/tsystems/mms/apm/performancesignature/model/CredProfilePair.java

```diff
@@ -70,12 +67,32 @@ public String getDisplayName() {
             return "";
         }
 
-        public ListBoxModel doFillCredentialsIdItems(@AncestorInPath final Project project) {
-            return new StandardListBoxModel()
-                    .withEmptySelection()
-                    .withMatching(instanceOf(UsernamePasswordCredentials.class),
-                            CredentialsProvider.lookupCredentials(
-                                    StandardUsernameCredentials.class, project, ACL.SYSTEM, Collections.<DomainRequirement>emptyList()));
+        public ListBoxModel doFillCredentialsIdItems(@AncestorInPath Jenkins context, @QueryParameter String credentialsId) {
```

Comment:
Ok then in that case remove `@AncestorInPath` and just use `Jenkins.getActiveInstance()` and `ACL.SYSTEM` directly as you are intending this to be a global setting